src/support_tests.cc

```diff
@@ -494,56 +542,67 @@ bool test_digest(bool print_all) {
 
   size_digest = (unsigned int) digest_output_byte_size("sha256");
   if (size_digest < 0) {
-    printf("failed 1 (%d)\n", size_digest);
+    printf("%s() error, line: %d, digest failed, %d\n",
```

Comment:
Same: Clarify in message that: `".... failed, size_digest=%d\n"`

Reply:
done